presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestStringStatisticsBuilder.java

```diff
@@ -325,7 +325,7 @@ private static void assertMinMaxValuesWithLimit(Slice expectedMin, Slice expecte
 
     private static void assertMinMaxValuesWithLimit(Slice expectedMin, Slice expectedMax, List<Slice> values, int limit, long expectedSum)
     {
-        checkArgument(values != null && values.size() > 0);
+        checkArgument(values.size() > 0);
```

Comment:
This is incorrect replacement

Reply:
This is deliberate. values== null causes a NullPointerException. values <= 0 causes an IllegalArgumentException